cosmos/x/evm/plugins/precompile/plugin.go

```diff
@@ -185,9 +182,6 @@ func (p *plugin) enableReentrancy(sdb vm.PolarisStateDB) {
 	// end precompile execution => stop emitting Cosmos event as Eth logs for now
 	cem := utils.MustGetAs[state.ControllableEventManager](sdkCtx.EventManager())
 	cem.EndPrecompileExecution()
-
-	// remove Cosmos gas consumption so gas is consumed only per OPCODE
-	p.sp.SetGasConfig(storetypes.GasConfig{}, storetypes.GasConfig{})
```

Comment:
Can we not just 0 the gas config on the given statedb here, would make much more sense